api/src/main/java/gov/cms/ab2d/api/security/JwtConfig.java

```diff
@@ -0,0 +1,25 @@
+package gov.cms.ab2d.api.security;
+
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Getter
+@Configuration
+public class JwtConfig {
```

Comment:
Let's keep only the values we actually need. I am not sure we need `secret` or `expiration` (the latter is set in the token generated by Okta). `uri` might not be needed, either.